README.md

```diff
@@ -173,9 +173,9 @@ Feel free to check the [issues page](../../issues/).
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
```

Comment:
- [ ] Since you have a new collaborator, to give proper credits, I suggest you update your `Authors` section to show the contacts of all contributors which in this case is two. In fact this good habit can save you from legal issues regarding plagiarism.